src/server/session.ts

```diff
@@ -1074,13 +1078,12 @@ namespace ts.server {
 
                     if (this.getPreferences(fileName).disableSuggestions) {
                         goNext();
+                        return;
                     }
-                    else {
-                        next.immediate(() => {
-                            this.suggestionCheck(fileName, project);
-                            goNext();
-                        });
-                    }
+                    next.immediate(() => {
+                        this.suggestionCheck(fileName, project);

```

Comment:
@sandersn Long-term, we might want to enable your just-do-it errors in this context.

Reply:
> Long-term, we might want to enable your just-do-it errors in this context.

Not familiar with this.